src/librustc_trans/trans/type_.rs

```diff
@@ -82,6 +82,10 @@ impl Type {
         ty!(llvm::LLVMInt64TypeInContext(ccx.llcx()))
     }
 
+    pub fn ix(ccx: &CrateContext, num_bits: u64) -> Type {
```

Comment:
Could you add a comment describing this, since the name `ix` isn't immediately obvious.